src/dashboards/videos/details.jsonnet

```diff
@@ -82,6 +83,15 @@ dashboard.new(
     refresh='time'
   )
 )
+.addTemplate(
+  template.custom(
+    name='VIEW_COUNT_THRESHOLD',
+    current='30',
+    label='View count threshold',
+    query='0,10,20,30,40,50,60',
+    refresh='time'
+  )
+)
```

Comment:
That is cool for us to look for the relevant threshold, but keep in mind that it's not relevant for our end-users :wink: (this should be a fixed value for the public dashboard).

Reply:
Ok) Good to know) Thanks)